Assets/Scripts/Editor/InventoryEditor.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor
```

Comment:
namespace already exists in animal pack

Reply:
**Important**: Delete or zip or `mv scripts scripts.disables` the Scripts folder in the LowPolyAnimatedAnimals pack